main.js

```diff
@@ -0,0 +1,78 @@
+// constant variable declare
+
+const addButton = document.getElementById('Add');
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const adder = document.getElementById('keeper');
+
+// setup the local storage
+
+if (localStorage.getItem('books') !== null) {
+  const getbook = JSON.parse(localStorage.getItem('books'));
```

Comment:
- [ ] I think it's better to create a code to present the data from the collection. Load the collection with the data from the local storage when starting. Your code will look more professional and with improved functionality.